### PR TITLE
Added Dutch Translations For Every Presence

### PR DESCRIPTION
**Pull request by DarkVillager**
I hope its good this time :)

### CODE REVIEW

File: websites/S/Scantrad/dist/metadata.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "service": "Scantrad",
   "description": {
     "en": "Scantrad is a free french website which allow you to read the latest scans of the moment.",
-    "fr": "Scantrad est un site internet français gratuit qui vous permet de lire les derniers scans du moment."
+    "fr": "Scantrad est un site internet français gratuit qui vous permet de lire les derniers scans du moment.",
+    "nl": "Scantrad is een gratis franse website waar u de laatste scans van het moment kunt lezen."
   },
   "url": "scantrad.net",
   "version": "1.0.2",
```
Comment (Bas950) [inline websites/S/Scantrad/dist/metadata.json:13]: bump version

File: websites/S/Scantrad/presence.ts [hunk 2]
```diff
@@ -10,20 +10,16 @@ presence.on("UpdateData", async () => {
...
+      "Reading " + document.querySelector(".tl-titre").textContent + ":";
     presenceData.state =
-      chapter.textContent.replace("Chapitre", "Chapter") +
+      chapter.options[chapter.selectedIndex].text.replace(
+        "Chapitre",
+        "Chapter"
+      ) +
       " | " +
       page.options[page.selectedIndex].text;
     presenceData.smallImageKey = "reading";
```
Comment (Bas950) [inline websites/S/Scantrad/presence.ts:25]: ??

File: websites/S/Snipes/dist/metadata.json [hunk 3]
```diff
@@ -23,19 +30,13 @@
...
-      "icon": "fas fa-compress-arrows-alt",
-      "value": true
-    },
-    {
-      "id": "showSmallImages",
-      "title": "Show Small Images",
+      "title": "Show buttons",
       "icon": "fas fa-compress-arrows-alt",
       "value": true
     }
```
Comment (Bas950) [inline websites/S/Snipes/dist/metadata.json:42]: ??

File: websites/S/Snipes/dist/metadata.json [hunk 4]
```diff
@@ -7,11 +7,18 @@
...
-  "version": "1.1.2",
+  "url": [
+    "www.snipes.com",
+    "www.snipes.at",
+    "www.snipes.fr",
+    "www.snipes.nl",
+    "www.snipes.it",
+    "www.snipes.es"
+  ],
+  "version": "1.0.2",
```
Comment (Bas950) [inline websites/S/Snipes/dist/metadata.json:21]: ??

File: websites/T/Trackmania.io/dist/metadata.json [hunk 5]
```diff
@@ -9,7 +9,7 @@
     "trackmania.io",
     "www.trackmania.io"
   ],
-  "version": "1.1.1",
+  "version": "1.0.3",
```
Comment (Bas950) [inline websites/T/Trackmania.io/dist/metadata.json:12]: ??

File: websites/T/TuMangaOnline/dist/metadata.json [hunk 6]
```diff
@@ -9,10 +9,10 @@
...
   "url": [
-    "lectortmo.com",
+    "tmofans.com",
     "tmocommunity.com"
   ],
-  "version": "1.2.6",
+  "version": "1.2.5",
   "logo": "https://i.imgur.com/1CYkKgl.png",
   "thumbnail": "https://i.imgur.com/SKXMHAG.png",
   "color": "#3366cc",
```
Comment (Bas950) [inline websites/T/TuMangaOnline/dist/metadata.json:18]: ???

File: websites/T/TugaStrikes/dist/metadata.json [hunk 7]
```diff
@@ -5,10 +5,11 @@
   },
   "service": "TugaStrikes",
   "description": {
-    "en": "TugaStrikes is a Gaming Community Founded in 2017 focusing on CS:GO"
+    "en": "TugaStrikes is a Gaming Community Founded in 2017 focusing on CS:GO",
+    "nl": "TugaStrikes is een gamingcommunity die is opgericht in 2017 en zich richt op CS: GO"
```
Comment (Bas950) [inline websites/T/TugaStrikes/dist/metadata.json:9]:
```suggestion
    "nl": "TugaStrikes is een gamingcommunity die is opgericht in 2017 en zich richt op CS:GO"
```

File: websites/V/Viki/dist/metadata.json [hunk 8]
```diff
@@ -15,7 +15,7 @@
     "blog.viki.com",
     "support.viki.com"
   ],
-  "version": "1.3.9",
+  "version": "1.3.7",
```
Comment (Bas950) [inline websites/V/Viki/dist/metadata.json:18]: ?

File: websites/W/WoolyPooly/dist/metadata.json [hunk 9]
```diff
@@ -9,7 +9,8 @@
     "zh_CN": "WoolyPooly是最有利可图的矿池之一。我们专注于PPLNS & SOLO奖励系统的最低佣金费率。",
     "ru": "WoolyPooly - один из самых прибыльных пулов, благодаря системам вознаграждения PPLNS и SOLO, а так же низким комиссиям.",
     "fa_IR": "WoolyPooly با تشکر از سیستم پاداش PPLNS & SOLO و هزینه های کمی برای تجارت ، یکی از استخرهای سودآور است",
-    "ga_IE": "Tá WoolyPooly ar cheann de na linnte is brabúsaí lenár bhfócas ar tháillí coimisiúin íosta do chórais luaíochta PPLNS & SOLO."
+    "ga_IE": "Tá WoolyPooly ar cheann de na linnte is brabúsaí lenár bhfócas ar tháillí coimisiúin íosta do chórais luaíochta PPLNS & SOLO.",
+    "nl": "WoolyPooly is een van de meest winstgevende pools met onze focus op minimale commissiekosten voor PPLNS & SOLO-beloningssystemen."
   },
   "url": "woolypooly.com",
   "version": "1.1.2",
```
Comment (Bas950) [inline websites/W/WoolyPooly/dist/metadata.json:16]: bump version

File: websites/Y/YouTube/dist/metadata.json [hunk 10]
```diff
@@ -31,7 +31,7 @@
     "www.youtube.com",
     "studio.youtube.com"
   ],
-  "version": "5.1.4",
+  "version": "5.1.2",
   "logo": "https://i.imgur.com/C9ZEmge.png",
```
Comment (Bas950) [inline websites/Y/YouTube/dist/metadata.json:35]: ?

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ririxidev** [comment]
also resolve conflicts

**Comment by ririxidev** [comment]
and I wanna tell you that by this you lost 8 files because of updating your fork.

**Comment by DarkVillager** [comment]
> and I wanna tell you that by this you lost 8 files because of updating your fork.

wdym?

**Comment by ririxidev** [comment]
Your first commit:
image
Now: 
image

**Comment by DarkVillager** [comment]
> Your first commit:
> image
> Now:
> image
> Well it is not 629 but still 2 files.

Thats intentional. Some presences were already translated and I by accident added them.

**Comment by ririxidev** [comment]
You are missing 16 translations in your fork:
image - it's screenshot from me running translator on your fork

**Comment by DarkVillager** [comment]
oh will fix asap

**Comment by ririxidev** [comment]
image

File: websites/S/Scantrad/presence.ts [hunk 11]
```diff
@@ -1,57 +0,0 @@
...
-    presenceData.state = "Home page";
-  }
-
-  if (presenceData.details == null) {
-    presence.setTrayTitle();
-    presence.setActivity();
-  } else {
-    presence.setActivity(presenceData);
-  }
-});
```
Comment (Bas950) [inline websites/S/Scantrad/presence.ts:57]: ???????????????

File: websites/S/Snipes/dist/metadata.json [hunk 12]
```diff
@@ -7,7 +7,8 @@
   "service": "Snipes",
   "description": {
     "en": "SNIPES is a store for popular brand clothes.",
-    "de": "SNIPES ist ein Handel für beliebte Markenklamotten."
+    "de": "SNIPES ist ein Handel für beliebte marken Klamotten.",
+    "nl": "SNIPES is een winkel voor kleding van populaire merken."
   },
   "url": "www.snipes.com",
   "regExp": "www[.]snipes(usa)?[.]([a-z]+)",
```
Comment (Bas950) [inline websites/S/Snipes/dist/metadata.json:14]: bump version

File: websites/T/TuMangaOnline/dist/metadata.json [hunk 13]
```diff
@@ -9,10 +9,10 @@
...
   },
   "url": [
-    "lectortmo.com",
+    "tmofans.com",
     "tmocommunity.com"
   ],
-  "version": "1.2.6",
+  "version": "1.2.7",
   "logo": "https://i.imgur.com/1CYkKgl.png",
   "thumbnail": "https://i.imgur.com/SKXMHAG.png",
```
Comment (Bas950) [inline websites/T/TuMangaOnline/dist/metadata.json:17]: undo

File: websites/V/Viki/dist/metadata.json [hunk 14]
```diff
@@ -15,7 +15,7 @@
     "blog.viki.com",
     "support.viki.com"
   ],
-  "version": "1.3.9",
+  "version": "1.3.10",
```
Comment (Bas950) [inline websites/V/Viki/dist/metadata.json:18]: undo

File: websites/Y/YouTube/dist/metadata.json [hunk 15]
```diff
@@ -87,4 +87,4 @@
       "value": true
     }
   ]
-}
+}
```
Comment (Bas950) [inline websites/Y/YouTube/dist/metadata.json:90]: undo

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by ririxidev — COMMENTED** [review]
waiting till bas will check it again XD
Also next time do it faster and you are still missing translations but do them later
image

File: websites/Y/YouTube/dist/metadata.json (hunk 15 above)
Comment (Bas950) [reply websites/Y/YouTube/dist/metadata.json:90]: did not solve the issue...

**Review by Bas950 — APPROVED** [review]

**Pull request merged**